stan/math/rev/fun/mdivide_left.hpp

```diff
@@ -179,6 +179,9 @@ inline Eigen::Matrix<var, R1, C2> mdivide_left(
 
   check_square("mdivide_left", "A", A);
   check_multiplicable("mdivide_left", "A", A, "b", b);
+  if (A.size() == 0) {
```

Comment:
[optional, because it's a minor point]

This should be `return {0,  b.cols()};` and `res` should be defined after this test.  Same elsewhere.

Reply:
Cool, I was going to do that, but then went back as it looked as I was moving unrelated code. I'll be happy to change these..